Components/space-utils/padding.css

```diff
@@ -0,0 +1,665 @@
+/* padding: 0.25rem */
+.p-0-25 {
+    padding: var(--space-0-25);
+}
+
+/* padding left and right 0.25rem */
+.px-0-25 {
+    padding-left: var(--space-0-25);
+    padding-right: var(--space-0-25);
+}
+
+/* padding left, right 0.25rem */
+.pl-0-25 {
+    padding-left: var(--space-0-25);
+}
+
+.pr-0-25 {
+    padding-right: var(--space-0-25);
+}
+
+/* padding top and bottom 0.25rem */
+.py-0-25 {
+    padding-top: var(--space-0-25);
+    padding-bottom: var(--space-0-25);
+}
+
+/* padding top, bottom 0.25rem */
+.pt-0-25 {
+    padding-top: var(--space-0-25);
+}
+
+.pb-0-25 {
+    padding-bottom: var(--space-0-25);
+}
+
+/* End of padding 0.25rem values */
+
+/* padding: 0.5rem */
+.p-0-5 {
+    padding: var(--space-0-5);
+}
+
+/* padding left and right 0.5rem */
+.px-0-5 {
+    padding-left: var(--space-0-5);
+    padding-right: var(--space-0-5);
```

Comment:
Here use shorthand like padding: 0 var(--space-0-5)

Reply:
I did not use shorthand because the user might want to add a separate spacing value for the top and bottom. Hence I did not add 0. Also, in the reset, I've already added 0.